home/views/authentication.py

```diff
@@ -26,7 +26,8 @@ def get(self, request):
         context = {
             "login_form": LoginForm(),
             "register_form": RegisterForm(),
-            "password_reset_form": ForgotPasswordForm()
+            "password_reset_form": ForgotPasswordForm(),
+            "next": request.GET.get("next")
```

Comment:
careful: if someone navigates straight to https://planetterp.com/login, this will be `None`. `request.GET.get("next", "/profile")` is better here.